### PR TITLE
Reducing time in DR integration tests

### PR DESCRIPTION
**Pull request by ManishKumarKeshri**
**Change log description**  
The PR speeds up the data recovery integration tests.

**Purpose of the change**  
Time reduction in running tests.

**What the code does**  
- Writes lesser number of events to verify recovery. 
- Creates just one stream to write events and read back.
- Removed redundant immediate read after writing the events, as the final read is enough to verify the successfulness of the test.
- Makes flushing of Tier1 to the long term storage faster using updated writer config.

**How to verify it**  
Build shall pass.

### CODE REVIEW

File: test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java [hunk 1]
```diff
@@ -375,17 +377,18 @@ public void close() throws Exception {
...
+     *  2. Writes 10 events.
+     *  3. Waits for Tier1 to be entirely flushed to the long term storage.
      *  4. Shuts down the controller, segment store and bookeeper/zookeeper.
-     *  5. Deletes container metadata segment and its attribute segment from the old LTS.
-     *  5. Starts just one debug segment container using a new bookeeper/zookeeper and the old LTS.
-     *  6. Re-creates the container metadata segment in Tier1 and let's it flushed to the LTS.
-     *  7. Starts segment store and controller.
-     *  8. Reads all 600 events again.
+     *  5. Creates back up of container metadata segment and its attribute segment from the old LTS before deleting them.
+     *  6. Starts just one debug segment container using a new bookeeper/zookeeper and the old LTS.
```
Comment (andreipaduroiu) [inline test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java:384]: "Just don't say just". :)

File: test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java [hunk 2]
```diff
@@ -375,17 +377,18 @@ public void close() throws Exception {
      * debug segment container.
      *  What test does, step by step:
      *  1. Starts Pravega locally with just one segment container.
-     *  2. Writes 300 events to two different segments.
-     *  3. Waits for all segments created to be flushed to the long term storage.
+     *  2. Writes 10 events.
+     *  3. Waits for Tier1 to be entirely flushed to the long term storage.
```
[the remark this answers is not in the document]
Comment (andreipaduroiu) [reply test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java:381]: and in the other tests javadoc as well.

File: test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java [hunk 3]
```diff
@@ -123,9 +124,9 @@
     private static final Duration TRANSACTION_TIMEOUT = Duration.ofMillis(10000);
 
     /**
-     * Write 300 events to different segments.
+     * Write 10 events to verify recovery.
      */
-    private static final int TOTAL_NUM_EVENTS = 300;
+    private static final int TOTAL_NUM_EVENTS = 10;
```
[the remark this answers is not in the document]
Comment (ManishKumarKeshri) [reply test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java]: Made it to 300.

File: test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java [hunk 4]
```diff
@@ -375,17 +377,18 @@ public void close() throws Exception {
      * debug segment container.
      *  What test does, step by step:
      *  1. Starts Pravega locally with just one segment container.
-     *  2. Writes 300 events to two different segments.
-     *  3. Waits for all segments created to be flushed to the long term storage.
+     *  2. Writes 10 events.
```
[the remark this answers is not in the document]
Comment (ManishKumarKeshri) [reply test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java]: Done.

File: test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ManishKumarKeshri) [reply test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java]: Done.

File: test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java [hunk 5]
```diff
@@ -375,17 +377,18 @@ public void close() throws Exception {
...
+     *  3. Waits for Tier1 to be entirely flushed to the long term storage.
      *  4. Shuts down the controller, segment store and bookeeper/zookeeper.
-     *  5. Deletes container metadata segment and its attribute segment from the old LTS.
-     *  5. Starts just one debug segment container using a new bookeeper/zookeeper and the old LTS.
-     *  6. Re-creates the container metadata segment in Tier1 and let's it flushed to the LTS.
-     *  7. Starts segment store and controller.
-     *  8. Reads all 600 events again.
+     *  5. Creates back up of container metadata segment and its attribute segment from the old LTS before deleting them.
```
[the remark this answers is not in the document]
Comment (ManishKumarKeshri) [reply test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java]: Fixed.

**Review by andreipaduroiu — APPROVED** [review]

**Pull request merged**